internal/controller/ingress/ippolicy_controller.go

```diff
@@ -135,6 +135,12 @@ func (r *IPPolicyReconciler) update(ctx context.Context, policy *ingressv1alpha1
 		return r.controller.ReconcileStatus(ctx, policy, err)
 	}
 
+	// If we found the remote policy, set the Created condition to true
+	// if it was not already set.
+	if !hasIPPolicyCreatedCondition(policy) {
+		setIPPolicyCreatedCondition(policy, true, ReasonIPPolicyCreated, "IP Policy already exists")
+	}
```

Comment:
After talking @alex-bezek, he pointed out that we probably don't even need the has condition check. The controller-runtime helper for this should gracefully handle that. 👍